kostyor/db/api.py

```diff
@@ -54,6 +52,26 @@ def create_discovery_method(method):
 
 def create_cluster_upgrade(cluster_id, to_version):
     cluster = db_session.query(models.Cluster).get(cluster_id)
+
+    if cluster is None:
+        raise exceptions.ClusterNotFound(
+            'Cluster "%s" not found.' % cluster_id)
+
+    if cluster.version == constants.UNKNOWN:
+        raise exceptions.ClusterVersionIsUnknown('Cluster version is unknown')
+
+    if cluster.status == constants.UPGRADE_IN_PROGRESS:
+        raise exceptions.UpgradeIsInProgress(
+            'Cluster %s already has an upgrade in progress.' % cluster_id)
```

Comment:
@sc68cal What do you mean here? I don't use the same check in `resources/upgrades.py`. It's maybe exist within `rest_api.py`, but I think we're moving toward getting rid of them. They are here just because `Kostyor-cli` doesn't use new endpoints yet.